packages/process/test/exec.test.ts

```diff
@@ -158,26 +181,166 @@ describe('exec', () => {
   // in windows at all.
   if (process.platform !== 'win32') {
     describe('when the `shell` option is true', () => {
-      it('lets the shell do all of the shellword parsing', function*() {
+      it('lets the shell do all of the shellword parsing', function* () {
         let proc = exec('echo "first" | echo "second"', {
-          shell: true
+          shell: true,
         });
         let { stdout }: ProcessResult = yield proc.expect();
 
-        expect(stdout).toEqual("second\n");
-      })
+        expect(stdout).toEqual('second\n');
+      });
     });
   }
 
   describe('when the `shell` option is `false`', () => {
-    it('automatically parses the command argumens using shellwords', function*() {
+    it('automatically parses the command arguments using shellwords', function* () {
       let proc = exec('echo "first" | echo "second"', {
-        shell: false
+        shell: false,
       });
       let { stdout }: ProcessResult = yield proc.expect();
 
-      expect(stdout).toEqual("first | echo second\n");
-    })
+      expect(stdout).toEqual('first | echo second\n');
+    });
   });
 
+  describe('handles env vars', () => {
+    describe('env as option - shell: bash', () => {
+      let shell = 'bash';
+
+      it('echo env', function* () {
+        let proc = exec('echo $EFFECTION_TEST_ENV_VAL', {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          //@ts-ignore
+          shell,
+          env: { EFFECTION_TEST_ENV_VAL: 'boop' },
+        });
+        let { stdout, code }: ProcessResult = yield proc.expect();
+
+        expect(stdout).toEqual('boop\n');
+        expect(code).toBe(0);
+      });
+
+      it('echo curly env', function* () {
+        let proc = exec('echo ${EFFECTION_TEST_ENV_VAL}', {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          //@ts-ignore
+          shell,
+          env: { EFFECTION_TEST_ENV_VAL: 'boop' },
+        });
+        let { stdout, code }: ProcessResult = yield proc.expect();
+
+        expect(stdout).toEqual('boop\n');
+        expect(code).toBe(0);
+      });
+    });
+
+    describe('env as option - shell: true', () => {
+      let shell = true;
+
+      it('echo env', function* () {
```

Comment:
tests should read as complete sentences starting with the word `it` e.g. "it can echo a passed in environment variable"